platform-controller/src/test/java/org/hobbit/controller/docker/VersionTagIdentificationTest.java

```diff
@@ -21,12 +21,18 @@
 import java.util.Collection;
 import java.util.List;
 
+import org.hobbit.controller.containers.docker.ContainerManagerImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.